ui/tests/acceptance/topology-test.js

```diff
@@ -56,8 +56,12 @@ module('Acceptance | topology', function(hooks) {
 
     await Topology.visit();
 
-    await Topology.viz.datacenters[0].nodes[0].memoryRects[0].select();
-    assert.equal(Topology.infoPanelTitle, 'Allocation Details');
+    if (Topology.viz.datacenters[0].nodes[0].isEmpty) {
+      assert.expect(0);
```

Comment:
I don't like that whether or not this test has assertions is at the whim of a faker seed, but I won't make you fix that. I'll get to it with my ongoing to top viz stuff.